homework/index.js

```diff
@@ -29,8 +29,32 @@
     return elem;
   }
 
-  function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+  function renderRepoDetails(repo, table) {
+    const RepositoryTr = createAndAppend('tr', table);
```

Comment:
Remember that variable names should be in camelCase (start with a lowercase letter).